source/deployment/metrics.rst

```diff
@@ -0,0 +1,173 @@
+Performance Monitoring (E20) - Beta
+===============================
+
+*Available in Enterprise Edition E20*.
+
+Performance monitoring support enables a Mattermost server to track system health for large Enterprise deployments through integrations with `Prometheus <https://prometheus.io/>`_ and `Grafana <http://grafana.org/>`_. 
+
+The integration supports data collection from several Mattermost servers, particularly useful if you're running Mattermost `in high availability mode <https://docs.mattermost.com/deployment/cluster.html>`_.
+
+Deployment Guide
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Details on integrating your Mattermost server with Prometheus and Grafana.
+
+Installing Prometheus
+------------------------------------------------
+
+1 - `Download a precompiled binary for Prometheus <https://prometheus.io/download/>`_. Binaries are provided for many popular distributions, including Darwin, Linux and Windows.
+
+For install instructions, see `Prometheus install guides <https://prometheus.io/docs/introduction/getting_started/>`_
+
+2 - The following settings are recommended in the Prometheus configuration file named ``prometheus.yml``:
+ 
+.. code:: yaml
+
+    # my global config
+    global:
+      scrape_interval:     60s # By default, scrape targets every 15 seconds.
+      evaluation_interval: 60s # By default, scrape targets every 15 seconds.
+      # scrape_timeout is set to the global default (10s).
+
+      # Attach these labels to any time series or alerts when communicating with
+      # external systems (federation, remote storage, Alertmanager).
+      external_labels:
+          monitor: 'mattermost-monitor'
+
+    # Load rules once and periodically evaluate them according to the global 'evaluation_interval'.
+    rule_files:
+      # - "first.rules"
+      # - "second.rules"
+
+    # A scrape configuration containing exactly one endpoint to scrape:
+    # Here it's Prometheus itself.
+    scrape_configs:
+      # The job name is added as a label `job=<job_name>` to any timeseries scraped from this config.
+      - job_name: 'prometheus'
+
+        # Override the global default and scrape targets from this job every 5 seconds.
+        # scrape_interval: 5s
+
+        # metrics_path defaults to '/metrics'
+        # scheme defaults to 'http'.
+
+        static_configs:
+          - targets: ["<hostname1>:<port>", "<hostname2>:<port>"]
+  }
+
+3 - Enable performance monitoring in the Mattermost System Console and specify the listen address. See more detail in our `configuration settings documentation <https://docs.mattermost.com/administration/config-settings.html#performance-monitoring-beta>`_.
+
+.. image:: ../images/perf_monitoring_system_console.png
```

Comment:
I tried doing
```
.. image:: ../images/perf_monitoring_system_console.png
  :width: 100px
```

and
```
.. image:: ../images/perf_monitoring_system_console.png
  :scale: 50%
```

```
.. image:: ../images/perf_monitoring_system_console.png
  :height: 10px
```

etc. but the image stayed the same size. Any tips?